### ISSUE TITLE
Namespacing for better overview

### ISSUE THREAD

**Issue by byteSamurai** [kubeless/vscode-kubeless#6]
Compared to openwhisks vscode solution I appreciate the possibility to get the logs directly. Very nice, but wouldn't it be more user friendly to declare some "cli namespace"

For example instead of:
```
Deploy Function
Call Function
Get Function Logs
Delete Function
```
my suggestion would be using a keyword like kubeless, k6s, kbls...something like that
```
kbls deploy
kbls call
kbls get-log
kbls delete
```

What do you think?

**Comment by andresmgot** [comment]
Yes, it's true that it would be better to specify somehow that the commands are related to `kubeless` but all the commands I see have a full title like `Git: Clone repository` so I would use:

```
Kubeless: Deploy Function
Kubeless: Call Function
Kubeless: Get Function Logs
Kubeless: Delete Function
```

would that work for you?
